xbmc/guilib/GUITexture.cpp

```diff
@@ -190,13 +190,25 @@ void CGUITexture::Render()
   Begin(color);
 
   // compute the texture coordinates
-  float u1, u2, u3, v1, v2, v3;
-  u1 = m_info.border.x1;
-  u2 = m_frameWidth - m_info.border.x2;
-  u3 = m_frameWidth;
-  v1 = m_info.border.y1;
-  v2 = m_frameHeight - m_info.border.y2;
-  v3 = m_frameHeight;
+  float u1, u2, u3, v1, v2, v3, du1, du2, du3, dv1, dv2, dv3;
```

Comment:
I know it's the pattern but we typically declare when used and single line per declaration

Reply:
I've done that in the [second](https://github.com/xbmc/xbmc/pull/21552/commits/45dcb227475e5c5ff6f601e9dabb91bda794a24c) commit.